marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/smart-mastering/entity-matching/test-data/entities/Customer_V1.entity.json

```diff
@@ -0,0 +1,95 @@
+{
+  "info": {
+    "title": "Customer_V1",
+    "version": "0.0.1",
+    "baseUri": "http://example.org/"
+  },
+  "definitions": {
+    "Customer_V1": {
```

Comment:
I recommend slimming this down to the bare minimum needed to produce the issue. Otherwise, there's an implication that all these properties are needed for the test. Also, don't you need a property with an underscore in its name too?